packages/driver/test/cypress/integration/commands/assertions_spec.js

```diff
@@ -2350,7 +2350,7 @@ describe('src/cy/commands/assertions', () => {
 
     context('attr', () => {
       beforeEach(function () {
-        this.$div = $('<div foo=\'bar\'>foo</div>')
+        this.$div = $('<div foo=\'bar\' fizz=\'buzz\'>foo</div>')
```

Comment:
You can use double quotes here - these escaped single quote `\'` are just an artifact of converting from CoffeeScript. I'd prefer double quotes.